restapi/stories.go

```diff
@@ -46,6 +52,16 @@ func (api api) ReturnStoryByID(w http.ResponseWriter, r *http.Request) render.Re
 	return rest.JSONStatusOK(story)
 }
 
+func (api api) randomizePinLocation(city string) (latitude float64, longitude float64, err error) {
```

Comment:
feels kinda icky that this is an `api` struct method, but I guess it's fine

Reply:
moved it